p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -2690,7 +2725,7 @@ The ```?:``` expression behaves as in C, e.g.:
 (x == 0) ? e0 : e1;
 ~ End P4Example
 
-The first argument is boolean, and the second and third arguments must
+The first argument is Boolean, and the second and third arguments must
```

Comment:
@mbudiu-vmw Andy Keep had already lowercased all occurrences of the two-element type that shall not be named. I don't care which way we go, but let's not keep editing over each other.